migration/package.json

```diff
@@ -7,17 +7,16 @@
     "start:dev": "NODE_ENV=dev tsx src/index.ts"
   },
   "dependencies": {
-    "config": "^3.3.8",
     "express": "^4.18.1",
     "firebase": "^9.10.0",
     "got": "^12.5.2",
     "pg": "^8.8.0",
     "typeorm": "^0.3.9"
   },
   "devDependencies": {
-    "@types/config": "^3.3.0",
     "@types/express": "^4.17.14",
     "@types/node": "^18.7.18",
+    "envsafe": "^2.0.3",
```

Comment:
needs to be a normal dependency since its used in code

Reply:
man i full felt smart putting it in as a dev dep :'(